adminSiteClient/EditorMapTab.tsx

```diff
@@ -88,6 +92,31 @@ class TimelineSection extends React.Component<{ mapConfig: MapConfig }> {
         this.props.mapConfig.timeTolerance = tolerance
     }
 
+    get timeToleranceStrategyOptions(): {
```

Comment:
This is still called `timeToleranceStrategy`, whereas everywhere else it's now just called `toleranceStrategy`.